internal/server/db/cluster/warnings.mapper.go

```diff
@@ -3,6 +3,7 @@
 package cluster
 
 // The code below was generated by incus-generate - DO NOT EDIT!
+//lint:file-ignore U1000,SA4006 Ignore staticcheck
```

Comment:
As suggested by the line above it, your change will get deleted the next time we run `make update-schema`.

I also don't think that it's right to ignore all lint on those files because they are auto-generated. Issues in those files are still issues, they just need to be fixed in the generator code rather than in the files themselves.